client/components/nps/index.js

```diff
@@ -55,15 +54,20 @@ const Nps = ( {
 	return (
 		<>
 			<div className="crowdsignal-forms-nps" style={ style }>
-				<h3 className="crowdsignal-forms-nps__question" style={ { whiteSpace: 'pre-wrap' } }>
-					{ decodeEntities( questionText ).split( '<br>' ).join( '\n' ) }
+				<h3
+					className="crowdsignal-forms-nps__question"
+					style={ { whiteSpace: 'pre-wrap' } }
+				>
+					{ decodeEntities( questionText )
+						.split( '<br>' )
+						.join( '\n' ) }
 				</h3>
 
 				<button
 					className="crowdsignal-forms-nps__close-button"
 					onClick={ onClose }
 				>
-					<Icon icon="no-alt" />
+					X
```

Comment:
Suggest using ✖ or ✕ instead, which are a bit more visually distinct from the letter X.

Alternatively, you could embed a tiny SVG here; in general it's a bad practice to do SVG-in-JS, but if it's something tiny that you're using in a single spot, that's fine.